### ISSUE TITLE
Add localisation for tab-titles on add-switch chain dapps panel

### ISSUE THREAD

**Issue by Pavneet-Sing** [brave/brave-browser#22672]
## Description 

[Network](https://github.com/brave/brave-core/blob/master/components/brave_wallet_ui/components/extension/allow-add-change-network-panel/index.tsx#L91) and [Details](https://github.com/brave/brave-core/blob/master/components/brave_wallet_ui/components/extension/allow-add-change-network-panel/index.tsx#L96) title should be localize.

## Steps to Reproduce

   1. Change language
   2. Trigger an add or a switch DApps request to verify the tab titles on panel

## Actual result:
No localisation 

## Expected result:
Should localize tab titles 

## Reproduces how often: 
easy

**Comment by Pavneet-Sing** [comment]
Putting it back to backlog as it's a coding change i.e. 
[`text='Network'`](https://github.com/brave/brave-core/blob/master/components/brave_wallet_ui/components/extension/allow-add-change-network-panel/index.tsx#L91)

should be replaced with something like

`text='getLocale('braveWalletNetwork')`
